vendor/clink.lua

```diff
@@ -59,6 +63,28 @@ local function get_folder_name(path)
 end
 
 
+---
+-- Forward/backward compatibility for Clink asynchronous prompt filtering.
+-- With Clink v1.2.10 and higher this lets git status run in the background and
+-- refresh the prompt when it finishes, to eliminate waits in large git repos.
+---
+local io_popenyield
+local clink_promptcoroutine
+local cached_info = {}
+if clink.promptcoroutine and io.popenyield then
+    io_popenyield = io.popenyield
+    clink_promptcoroutine = clink.promptcoroutine
+    -- Uncommenting this will override the cmderGitStatusOptIn and always show
+    -- git status when Clink is able to run it in the background.
+    --cmderForceAsyncGitStatus = true
```

Comment:
could this be a config setting in the https://github.com/cmderdev/cmder/blob/master/vendor/cmder_prompt_config.lua.default

Reply:
Sure, I'll add it!  I wasn't sure whether/where/how to make it configurable.  :)

Reply:
Done.  Do the name and description seem reasonable?

Reply:
@chrisant996 Why set a setting that just controls setting another setting?  Why not just set the setting?

Reply:
The intent was to decouple the user-configurable setting from an internal programmatic variable that the Cmder code may set to override the user-configurable setting if necessary.

But I can remove that, no problem.

Reply:
It is crucial for correct functionality:

- `prompt_overrideGitStatusOptIn` is a user-configurable setting.
- `cmderForceAsyncGitStatus` is a programmatically set variable, and it must not respect the user-configurable setting when using a version of Clink that does not support async prompt filtering.

I can make `cmderForceAsyncGitStatus` be a `local` variable, so that it doesn't feel like a setting.

Reply:
Cool it was just a question.  I was sure you had a reason to do it.  Glad its better now.